crates/sui-json-rpc/src/read_api.rs

```diff
@@ -121,6 +123,52 @@ impl SuiRpcModule for ReadApi {
 
 #[async_trait]
 impl RpcFullNodeReadApiServer for FullNodeApi {
+    async fn get_move_function_args_tagging(
+        &self,
+        package: ObjectID,
+        module: String,
+        function: String,
+    ) -> RpcResult<Vec<String>> {
+        let object_read = self
+            .state
+            .get_object_read(&package)
+            .await
+            .map_err(|e| anyhow!("{e}"))?;
+
+        let normalized = match object_read {
+            ObjectRead::Exists(_obj_ref, object, _layout) => match object.data {
+                Data::Package(p) => p.normalize().map_err(|e| anyhow!("{e}")),
+                _ => Err(anyhow!("Object is not a package with ID {}", package)),
+            },
+            _ => Err(anyhow!("Package object does not exist with ID {}", package)),
+        }?;
+
+        let identifier = Identifier::new(function.as_str()).map_err(|e| anyhow!("{e}"))?;
+        let parameters = normalized.get(&module).and_then(|m| {
+            m.exposed_functions
+                .get(&identifier)
+                .map(|f| f.parameters.clone())
+        });
+
+        Ok(match parameters {
+            Some(parameters) => Ok(parameters
+                .iter()
+                .map(|p| match p {
+                    Type::Struct {
+                        address: _,
+                        module: _,
+                        name: _,
+                        type_arguments: _,
+                    } => "TRANSFER".into(),
```

Comment:
Maybe use something like "FULL ACCESS" here? Since having a struct here in the parameter means the function has full access of the object and can read, write, transfer, destroy, etc.

Reply:
"BY VALUE" is also a term we use elsewhere.

But I think it is hard to capture the write terminology here:
- "TRANSFER" has issues since it might be destroyed 
- "FULL ACCESS" feels like it is somehow just read+write? or somehow doesn't really convey to me that it might be destroyed